cluster-autoscaler/FAQ.md

```diff
@@ -182,6 +183,29 @@ No. CPU-based (or any metric-based) cluster/node group autoscalers, like
 [GCE Instance Group Autoscaler](https://cloud.google.com/compute/docs/autoscaler/), are NOT compatible with CA.
 They are also not particularly suited to use with Kubernetes in general.
 
+
+### How Cluster Autoscaler works with Pod Priority and Preemption?
+
+Pod Priority and Preemption feature enables K8S users to schedule pods based on priorities if there is not enough resources.
```

Comment:
Add information about from which version will this be handled in this way and how it was handled  before.

Reply:
done